compiler/resolution/functionResolution.cpp

```diff
@@ -11247,6 +11247,21 @@ static void resolveExterns()
   }
 }
 
+// Returns 'true' if the type 'at' was resolved.
+static bool maybeForceResolveAggregateType(AggregateType* at) {
+  if (at == nullptr) return false;
+  for_SymbolSymExprs(use, at->symbol) {
+    if (use->inTree()) {
+      at->resolveConcreteType();
+      for_fields(field, at) {
+        auto t = field->typeInfo();
+        INT_ASSERT(t != dtUnknown);
+      }
+      return true;
```

Comment:
The returned bool doesn't seem to be used, so should this return nothing?

Reply:
Is there a reason we can't simply just call ``resolveConcreteType``? Does it matter if it's actually used or not?

Reply:
I was trying to avoid resolving `c_ptr` if possible, since that resolves some module code.